Git Testing/Scenes/Details/View/PictureDetailVC.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  PictureDetailViewController.swift
+//  Git Testing
+//
+//  Created by BrainX Technologies on 14/01/2021.
+//
+
+import UIKit
+
+class PictureDetailVC: UIViewController {
+    //MARK:- IBOUTLETS
+    @IBOutlet weak var productImageView: UIImageView!
+    
+    //MARK:- PROPERTIES
+    var picture: Picture?
+    var viewModel: PictureDetailViewModel?
+    
+    //MARK:- View Controller Lifecycle Function
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        guard picture != nil else {return}
+        viewModel = PictureDetailViewModel(picture: picture)
```

Comment:
This should be done outside the VC and pass the viewModel to the VC. Picture shouldn't e there.